<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tw_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Currency Pair Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SMA</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Main</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
